src/containers/list.js

```diff
@@ -19,71 +21,36 @@ const Container = styled.div`
   position: relative;
 `;
 
-export const buildAddressString = provider => {
+export const buildAddressString = (provider) => {
   const ADDRESS_1 = provider["provider address 1"];
   const ADDRESS_2 = provider["provider address 2"];
   const COUNTY = provider["provider county"];
   const TOWN = provider["provider town/city"];
   const POSTCODE = provider["provider postcode"];
 
   const addressArray = [ADDRESS_1, ADDRESS_2, COUNTY, TOWN, POSTCODE].filter(
-    parts => parts !== "Not Available" && parts
+    (parts) => parts !== "Not Available" && parts
   );
   return addressArray.join(", ");
 };
 
 const ListView = () => {
-  const { isGeolocationAvailable, coords } = useContext(GeoContext);
+  const {data, selectedLocation, setSelectedLocation, locations} = useContext(DataContext)
```

Comment:
I'm wondering if this, and the same line in map.js, could be pulled up a level into the router component? This would mean we're only fetching the data once rather than per component

Reply:
yeah sounds good, thanks